packages/dds/tree/prettier.config.cjs

```diff
@@ -0,0 +1,40 @@
+// TODO: Use shared config in build-common once version with config has been published.
+module.exports = {
+    printWidth: 100,
+    quoteProps: "consistent",
+    semi: true,
+    singleQuote: false,
+    trailingComma: "all",
+    overrides: [
+        {
+            files: "lerna.json",
+            options: {
+                printWidth: 50,
+                tabWidth: 2,
+            },
+        },
+        {
+            files: "tsconfig*.json",
+            options: {
+                parser: "json5",
+                tabWidth: 2,
+                trailingComma: "all",
+                quoteProps: "preserve",
+            },
+        },
+        {
+            files: "*.json",
+            options: {
+                tabWidth: 2,
+                trailingComma: "all",
+                quoteProps: "preserve",
+            },
+        },
+        {
+            files: "*.{js,ts}",
+            options: {
+                useTabs: true,
```

Comment:
While I personally prefer tabs to spaces, this repo seems to mainly use 4 space indent, so picking tabs here seems questionable.

Reply:
Per our .editorconfig, we use 2 spaces in 'package.json' and *.yml.  4 spaces everywhere else.

```
# Don't go up further looking for more .editorconfig files
root = true

[*]
charset = utf-8
end_of_line = lf
indent_size = 4
indent_style = space
insert_final_newline = true
trim_trailing_whitespace = true

# C-style doc comments
block_comment_start = /*
block_comment = *
block_comment_end = */

[*.{js,jsx,ts,tsx}]
indent_size = 4

[{*.yml,package.json}]
indent_size = 2
```